UI/src/view/TrafficSignLabel.js

```diff
@@ -25,47 +25,116 @@
+        {signValue: [14], image: 'images/traffic-signs/noPowerDrivenVehiclesSign.png'},
+        {signValue: [15], image: 'images/traffic-signs/noLorriesSign.png'},
+        {signValue: [16], image: 'images/traffic-signs/noVehicleCombinationsSign.png'},
+        {signValue: [17], image: 'images/traffic-signs/noTractorSign.png'},
+        {signValue: [18], image: 'images/traffic-signs/noMotorCycleSign.png'},
+        {signValue: [19], image: 'images/traffic-signs/noMotorSledgesSign.png'},
+        {signValue: [20], image: 'images/traffic-signs/noDangerousGoodsSign.png'},
+        {signValue: [21], image: 'images/traffic-signs/noBusSign.png'},
+        {signValue: [22], image: 'images/traffic-signs/noMopedsSign.png'},
+        {signValue: [23], image: 'images/traffic-signs/noCycleSign.png'},
+        {signValue: [24], image: 'images/traffic-signs/noPedestrianSign.png'},
+        {signValue: [25], image: 'images/traffic-signs/noPedestrianOrCycleSign.png'},
+        {signValue: [26], image: 'images/traffic-signs/noHorsesSign.png'},
+        {signValue: [27], image: 'images/traffic-signs/noEntrySign.png'},
+        {signValue: [28], image: 'images/traffic-signs/overtakingProhibitedSign.png'},
+        {signValue: [29], image: 'images/traffic-signs/endOfOvertakingProhibitonSign.png'},
+        {signValue: [30], image: 'images/traffic-signs/maxWidthSign.png', validation: validateMaximumRestrictions, convertion: convertToMeters },
+        {signValue: [31], image: 'images/traffic-signs/maxHeightSign.png', validation: validateMaximumRestrictions, convertion: convertToMeters},
+        {signValue: [32], image: 'images/traffic-signs/totalWeightLimit.png', validation: validateMaximumRestrictions, offset: -15 - (counter * 30), convertion: convertToTons },
+        {signValue: [33], image: 'images/traffic-signs/trailerTruckWeightLimit.png', validation: validateMaximumRestrictions, offset: -10 - (counter * 30), convertion: convertToTons },
+        {signValue: [34], image: 'images/traffic-signs/axleWeightLimit.png', validation: validateMaximumRestrictions, offset: -18 - (counter * 30), convertion: convertToTons },
+        {signValue: [35], image: 'images/traffic-signs/bogieWeightLimit.png', validation: validateMaximumRestrictions, offset: -18 - (counter * 30), convertion: convertToTons },
+        {signValue: [36], image: 'images/traffic-signs/rightBendSign.png'},
+        {signValue: [37], image: 'images/traffic-signs/leftBendSign.png'},
+        {signValue: [38], image: 'images/traffic-signs/severalBendRightSign.png'},
+        {signValue: [39], image: 'images/traffic-signs/severalBendLeftSign.png'},
+        {signValue: [40], image: 'images/traffic-signs/dangerousDescentSign.png'},
+        {signValue: [41], image: 'images/traffic-signs/steepAscentSign.png'},
+        {signValue: [42], image: 'images/traffic-signs/unevenRoadSign.png'},
+        {signValue: [43], image: 'images/traffic-signs/childrenSign.png'}
+      ];
+
+      function find() {
```

Comment:
Instead of having a method here you can have it as `var` and perform the find just once and then use it in all the methods below.